contributed_definitions/NXibeam_column.nxdl.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXibeam_column" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
+    <doc>
+         Container for components of a focused-ion-beam (FIB) system.
+         
+         FIB capabilities turn especially scanning electron microscopes
+         into specimen preparation labs. FIB is a material preparation
+         technique whereby portions of the sample are illuminated with a
+         focused ion beam with controlled intensity so use ions with a 
+         high enough momentum to remove material in a controllable manner.
+         
+         The fact that an electron microscope with FIB capabilities has a 
+         second gun with own relevant control circuits, focusing lenses, and
+         other components warrants an own base class to group these components
+         and distinguish them from the lenses and components of the
+         electron column.
+         
+         For more details about the relevant physics and application examples
+         consult the literature, for example:
+         * [L. A. Giannuzzi et al.](https://doi.org/10.1007/b101190)
```

Comment:
blank lines before and after bullet list

Reply:
changed, also for all other enumerations and itemize lists